extensions/projection/README.md

```diff
@@ -9,30 +9,36 @@
 
-The field names defined herein should be added as fields in the Item Properties object. 
+The field names defined herein should be added as fields in the Item Properties object.
 
-When specified on an Item, the values are assumed to apply to all Assets in that Item.  For example, an Item may have 
-several related Assets each representing a band or layer for the Item, and which typically all use the same CRS, 
-e.g. a UTM Zone.  However, there may also be assets intended for display, like a preview image or thumbnail, that have 
-been reprojected to a different CRS, e.g., Web Mercator, to better accommodate that use case.  This case of differing 
-projections per Asset is not currently handled by this extension.
+When specified on an Item, the values are assumed to apply to all Assets in that Item. For example, an Item may have
+several related Assets each representing a band or layer for the Item, and which typically all use the same CRS,
+e.g. a UTM Zone. However, there may also be assets intended for display, like a preview image or thumbnail, that have
+been reprojected to a different CRS, e.g., Web Mercator, to better accommodate that use case. This case of differing
+projections per Asset is not currently handled by this extension. The exception to this is that `proj:shape` and
+`proj:transform` can be overridden at the asset level, where an asset has a different shape and transform from the
+default, which should be specified at the asset level, while those assets that use the defaults can remain unspecified.
 
 ## Examples
+
 - [Example Landsat8](examples/example-landsat8.json)
 
 ## Schema
+
 - [JSON Schema](json-schema/schema.json)
 
 ## Item Properties fields
 
 | Field Name       | Type                     | Description |
 | ---------------- | ------------------------ | ----------- |
-| proj:epsg        | integer\|null   | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
+| proj:epsg        | integer \|null  | **Required** [EPSG code](http://www.epsg-registry.org/) of the datasource |
 | proj:wkt2        | string \|null   | [WKT2](http://docs.opengeospatial.org/is/12-063r5/12-063r5.html) string representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
 | proj:projjson    | [PROJJSON Object](https://proj.org/usage/projjson.html) \|null   | PROJJSON object representing the Coordinate Reference System (CRS) that the `proj:geometry` and `proj:bbox` fields represent |
 | proj:geometry    | [Polygon Object](https://geojson.org/schema/Polygon.json)  | Defines the footprint of this Item. |
 | proj:bbox        | \[number]       | Bounding box of the Item in the asset CRS in 2 or 3 dimensions. |
 | proj:centroid    | Centroid Object | Coordinates representing the centroid of the Item in the asset CRS |
+| proj:shape       | \[number]       | Number of pixels in Y and X directions for the default grid |
```

Comment:
Below it's said that the shape consists of integers, so I'd propose to use [integer].

Reply:
👍